src/test/pybind/test_cephfs.py

```diff
@@ -831,6 +831,31 @@ def test_snapdiff(testdir):
     # remove directory
     purge_dir(b"/snapdiff_test");
 
+def test_blockdiff(testdir):
+    """
+    Tests the incremental sync syncs only the changed portions of a huge file.
+    """
+    cephfs.mkdir("/blockdiff_test", 0o755)
```

Comment:
I belive it would be nicer if the name indicates that it's a dir.  So perhaps `blockdiff_testdir`?